app/src/main/java/org/fossasia/susi/ai/skills/SkillsActivity.kt

```diff
@@ -147,22 +167,24 @@ class SkillsActivity : AppCompatActivity(), SkillFragmentCallback {
         }
     }
 
-    fun doSearch(query: String) {
+    //Used to perform search for the query entered by the user
+    fun doSearch(query: String): Boolean {
```

Comment:
Rename the function to tell what it's doing, `doSearch` sounds like a void function. What does the boolean returning signify. You have added a comment which does not explain this. The comment is obvious that this performs search, we can see that from the function name